libs/sdk-ui-tests-e2e/cypress/tools/api.ts

```diff
@@ -53,4 +53,52 @@ export class Api {
             req.headers["Authorization"] = `Bearer ${userToken}`;
         });
     }
+
+    postDrillDownStepAttributeDF = (drillFromAttr: string, drillToAttr?: string) => {
```

Comment:
what is it purpose? do it use to set up drill down?
name it more clearly, ex: setUpDrillDownAttribute

Reply:
is there any case that we don't need drillToAttr param?